webapp/packages/tests-runner/package.json

```diff
@@ -14,6 +14,8 @@
   },
   "dependencies": {
     "@cloudbeaver/core-di": "^0",
+    "@jest/globals": "^29",
+    "@testing-library/jest-dom": "^6",
     "@testing-library/react": "^16",
```

Comment:
isnt it all `devDependencies`?
including `"@testing-library/react": "^16",`

Reply:
for tests-runner it's not because it produces functions that actually depends on this, but other packages uses tests-runner as a devDependency